test/stableTRIStaking/transfer.test.ts

```diff
@@ -0,0 +1,313 @@
+    await this.rewardToken.burn(this.deployer.address, depositAmount);
+
+    expect(await this.tri.balanceOf(this.deployer.address)).to.equal(depositAmount);
+    await this.pTRI.connect(this.deployer).deposit(depositAmount);
+
+    expect(await this.tri.balanceOf(this.pTRI.address)).to.equal(depositAmount);
+    expect(await this.pTRI.balanceOf(this.deployer.address)).to.equal(depositAmount);
+
+    await this.pTRI.connect(this.deployer).approve(this.user1.address, depositAmount);
+    let deployerpTRIAllowance = await this.pTRI
+      .connect(this.deployer)
+      .allowance(this.deployer.address, this.user1.address);
+    expect(deployerpTRIAllowance).to.equal(depositAmount);
+
+    await this.pTRI.connect(this.user1).transferFrom(this.deployer.address, this.user1.address, depositAmount);
+    deployerpTRIAllowance = await this.pTRI.allowance(this.deployer.address, this.user1.address);
+    expect(deployerpTRIAllowance).to.equal(0);
+
+    expect(await this.rewardToken.balanceOf(this.deployer.address)).to.equal(0);
+    expect(await this.rewardToken.balanceOf(this.user1.address)).to.equal(0);
+    expect(await this.pTRI.balanceOf(this.deployer.address)).to.equal(0);
+    expect(await this.pTRI.balanceOf(this.user1.address)).to.equal(depositAmount);
+  });
+
+  it("should pre-claim zero before transfer if already claimed", async function () {
+    const depositAmount = 100;
+    await this.tri.mint(this.deployer.address, depositAmount);
+
+    expect(await this.tri.balanceOf(this.deployer.address)).to.equal(depositAmount);
+    await this.pTRI.connect(this.deployer).deposit(depositAmount);
+
+    expect(await this.tri.balanceOf(this.pTRI.address)).to.equal(depositAmount);
+    expect(await this.pTRI.balanceOf(this.deployer.address)).to.equal(depositAmount);
+
+    await this.rewardToken.transfer(this.pTRI.address, depositAmount);
+    expect(await this.rewardToken.balanceOf(this.deployer.address)).to.equal(0);
+
+    await this.pTRI.connect(this.deployer).transfer(this.user1.address, depositAmount);
+
+    expect(await this.rewardToken.balanceOf(this.deployer.address)).to.equal(100);
+    expect(await this.rewardToken.balanceOf(this.user1.address)).to.equal(0);
+    expect(await this.pTRI.balanceOf(this.deployer.address)).to.equal(0);
+    expect(await this.pTRI.balanceOf(this.user1.address)).to.equal(depositAmount);
+
+    await this.pTRI.connect(this.user1).transfer(this.deployer.address, depositAmount);
+
+    expect(await this.rewardToken.balanceOf(this.deployer.address)).to.equal(100);
+    expect(await this.rewardToken.balanceOf(this.user1.address)).to.equal(0);
+    expect(await this.rewardToken.balanceOf(this.pTRI.address)).to.equal(0);
+    expect(await this.pTRI.balanceOf(this.deployer.address)).to.equal(100);
+    expect(await this.pTRI.balanceOf(this.user1.address)).to.equal(0);
+  });
+
+  it("should claim before deposit and withdraw from escrow, escrow claims on withdrawal then user withdraws from pTRI", async function () {
```

Comment:
can you remove this test, since the next test is same 
only difference being we are also checking for allowance